src/Umbraco.Deploy.Contrib.Connectors/ValueConnectors/BlockEditorValueConnector.cs

```diff
@@ -8,20 +8,27 @@
 using Umbraco.Core.Logging;
 using Umbraco.Core.Models;
 using Umbraco.Core.Services;
+using Umbraco.Deploy.Connectors;
+using Umbraco.Deploy.Connectors.ValueConnectors;
 using Umbraco.Deploy.Connectors.ValueConnectors.Services;
+using Umbraco.Deploy.Core;
 
 namespace Umbraco.Deploy.Contrib.Connectors.ValueConnectors
 {
     /// <summary>
     /// A Deploy connector for BlockEditor based property editors (ie. BlockList)
     /// </summary>
-    public abstract class BlockEditorValueConnector : IValueConnector
+    public abstract class BlockEditorValueConnector : ValueConnectorBase
     {
         private readonly IContentTypeService _contentTypeService;
         private readonly Lazy<ValueConnectorCollection> _valueConnectorsLazy;
         private readonly ILogger _logger;
 
-        public virtual IEnumerable<string> PropertyEditorAliases => new[] { "Umbraco.BlockEditor" };
+        /// <inheritdoc />
+        public override IEnumerable<string> PropertyEditorAliases { get; } = new[]
+        {
+            "Umbraco.BlockEditor"
+        };
```

Comment:
There's no generic block editor (`Umbraco.BlockEditor` does not exist, only `Umbraco.BlockList` and soon `Umbraco.BlockGrid`) and this is an abstract class, so this should actually not implement/override the abstract property from the `ValueConnectorBase`...

Although removing this override is a breaking change, so is changing the base class to `ValueConnectorBase`, as the `ToArtifact()` and `FromArtifact()` methods without `IContextCache` are sealed and this previously declared them as virtual.

Reply:
This is now fixed in https://github.com/umbraco/Umbraco.Deploy.Contrib/pull/59/commits/928afaf0d6d009aca01a1cfb19577de57a1be8ff 👍🏻